src/main/java/wordsWithDoubleLetters/WordsWithDoubleLetters.java

```diff
@@ -0,0 +1,19 @@
+package wordsWithDoubleLetters;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class WordsWithDoubleLetters {
+    private static final Pattern pattern = Pattern.compile("\\w+");
```

Comment:
in Java code style constants are usually named UPPER_CASED: `PATTERN` :)

Reply:
![image](https://user-images.githubusercontent.com/6898003/85817410-901c0200-b776-11ea-8ddb-b97177570cd4.png)
:D

Reply:
> nit: in Java code style constants are usually named UPPER_CASED: `PATTERN` :)

fixed =)